src/sweeper/__main__.py

```diff
@@ -32,12 +32,15 @@ def main():
     '''Main entry point for program. Parse arguments and pass to sweeper modules.
     '''
     args = docopt(__doc__, version='1.0.0')
-
+    print(args)
```

Comment:
let's remove this

Reply:
I removed the print(args) and pushed up the changes.  let me know if you want any other changes.  if not, let's merge it and as steve said, get other folks testing it.